README.md

```diff
@@ -1,12 +1,12 @@
-# microbit-svg
+### microbit-svg
 
 This is an SVG drawing of the BBC micro:bit. It will render fine in a modern browser, and was designed in Inkscape.
 
 It's made of a number of SVG layers, one for each colour micro:bit and an example layer for the 'heart' shaped LEDs.
 
 If you'd like to make your own pattern of LEDs, it's easiest to duplicate the LEDsOn layer, then hide that layer and edit the new copy to match.
 
-# License
+### License
```

Comment:
You are right that this shoudn't be `h1` (`#`), but we should set this to `h2` (`##`) instead of `h3`.

```suggestion
## License
```